webrisk/database_test.go

```diff
@@ -511,8 +511,8 @@ func TestDatabaseUpdate(t *testing.T) {
 
 func TestDatabaseLookup(t *testing.T) {
 	var (
-		td0 = ThreatType(0)
-		td1 = ThreatType(1)
+		td0 = ThreatType(ThreatType_Unspecified)
```

Comment:
Not clear to me what `td0` and friends are for - can the values be used directly?

Reply:
done.